lib/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 include(GNUInstallDirs)
 
-add_library(mtkahypar SHARED libmtkahypar.cpp)
+add_library(mtkahypar libmtkahypar.cpp)
```

Comment:
I think here is the place where you want to use the cmake option `BUILD_SHARED_LIBS`?

Reply:
The `BUILD_SHARED_LIBS` options applies to all `add_library` instances, so in this case it should be a question of preference and I thought that it would be better to go with all the other option definitions.
Should I still move it to the subdir config?